platform/tailwind.config.js

```diff
@@ -82,11 +83,6 @@ module.exports = {
         '2xl': '6rem',
       },
     },
-    height: {
-      200: '400px',
-      240: '480px',
-      250: '500px',
-    },
   },
```

Comment:
Removed height config as it was breaking the inbuilt height config as shown in production
![image](https://user-images.githubusercontent.com/46527380/236196437-6b0086ba-37d9-4096-937b-132bb21ed8b4.png)